src/location_service/com/ubuntu/location/service/daemon.cpp

```diff
@@ -485,7 +507,8 @@ std::istream& location::service::operator>>(std::istream& in, location::service:
         {"is_online", location::service::Daemon::Cli::Property::is_online},
         {"does_satellite_based_positioning", location::service::Daemon::Cli::Property::does_satellite_based_positioning},
         {"does_report_wifi_and_cell_ids", location::service::Daemon::Cli::Property::does_report_wifi_and_cell_ids},
-        {"visible_space_vehicles", location::service::Daemon::Cli::Property::visible_space_vehicles}
+        {"visible_space_vehicles", location::service::Daemon::Cli::Property::visible_space_vehicles},
+        {"client_applications", location::service::Daemon::Cli::Property::client_applications},
```

Comment:
Would clients be able to use the dbus interface to inquire about other clients using location-service?

Reply:
No, this D-Bus interface is not accessible by clients (AppArmor blocks it).